src/rgw/rgw_object_lock.h

```diff
@@ -0,0 +1,219 @@
+#ifndef CEPH_RGW_OBJECT_LOCK_H
+#define CEPH_RGW_OBJECT_LOCK_H
+
+#include <string>
+#include "common/ceph_time.h"
+#include "common/iso_8601.h"
+#include "rgw_xml.h"
+using namespace std;
```

Comment:
in header files, please qualify types with `std::` instead of `using namespace std`